### ISSUE TITLE
Add support for accounts on lbry.tv

### ISSUE THREAD

**Issue by seanyesmunt** [lbryio/lbry-desktop#2479]
From notion:
# LBRYTV Flows v0.2

## Anonymous

1. If no session is present UI calls `internal-api/user/new` and `auth_token` is retrieved
2. UI forwards `auth_token` to lbryweb, a user record is created and an account is registered with the SDK

## Registration

1. `internal-api/user_email/new` method is called with user’s email as an argument
2. An email with a confirmation link is sent to the user
3. UI starts querying `internal-api/user_email/status`
4. User clicks the link in the confirmation email, calling `internal-api/user_email/confirm`
5. UI sends `auth_token` to lbryweb, a user is marked as registered

**Comment by tzarebczan** [comment]
Blocked by https://github.com/lbryio/lbrytv/pull/28

**Comment by kauffj** [comment]
@tzarebczan will open new ticket with remaining changes required
